File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/Border.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class Border {
    public static final String FAIR_NAME = "fair_name";
    public static final String FAIR = "fair";
    public static final String BORDER_NAME = "border_name";
    public static final String BORDER = "border";


    @SerializedName(FAIR_NAME)
    private String fair_name;

    @SerializedName(FAIR)
    private String fair;

    @SerializedName(BORDER_NAME)
    private String border_name;

    @SerializedName(BORDER)
    private String border;

    public String getFairName() {
        return fair_name;
    }

    public String getFair() {
        return fair;
    }

    public String getBorderName() {
        return border_name;
    }

    public String getBorder() {
        return border;
    }


}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/Border.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

object Border {
    const val FAIR_NAME = "fair_name"
    const val FAIR = "fair"
    const val BORDER_NAME = "border_name"
    const val BORDER = "border"

    @SerializedName(FAIR_NAME)
    val fairName: String? = null

    @SerializedName(FAIR)
    val fair: String? = null

    @SerializedName(BORDER_NAME)
    val borderName: String? = null

    @SerializedName(BORDER)
    val border: String? = null
}